### PR TITLE
4293 message modal

### PR DESCRIPTION
**Pull request by codyseibert**

### CODE REVIEW

File: web-client/src/presenter/actions/getUsersInSelectedSectionAction.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ export const getUsersInSelectedSectionAction = async ({
   applicationContext,
   props,
 }) => {
+  console.log(props);
```
Comment (matthopson) [inline web-client/src/presenter/actions/getUsersInSelectedSectionAction.js]: You wanna leave this?

File: web-client/src/presenter/actions/validateCreateCaseMessageAction.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { state } from 'cerebral';
+
+/**
+ * validates the create case message modal
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.path the cerebral path which contains the next path in the sequence (path of success or failure)
+ * @param {object} providers.props the cerebral props object
```
Comment (matthopson) [inline web-client/src/presenter/actions/validateCreateCaseMessageAction.js]: get, path

File: shared/src/business/useCases/messages/validateCreateCaseMessageInteractor.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+/**
+ * validateCreateCaseMessageInteractor
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.applicationContext the application context
+ * @param {object} providers.message the message data
+ * @returns {object} errors (null if no errors)
+ */
+exports.validateCreateCaseMessageInteractor = ({
```
Comment (rachaelparris) [inline shared/src/business/useCases/messages/validateCreateCaseMessageInteractor.js:11]: Missing tests

File: web-client/src/presenter/actions/validateCreateCaseMessageAction.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+ * @param {object} providers.applicationContext the application context needed for getting the validateForwardMessage use case
+ * @returns {object} path.success or path.error
+ */
+export const validateCreateCaseMessageAction = ({
```
Comment (rachaelparris) [inline web-client/src/presenter/actions/validateCreateCaseMessageAction.js:12]: Missing tests

**Review by rachaelparris — APPROVED** [review]

**Review by matthopson — APPROVED** [review]

File: web-client/src/presenter/sequences/updateCreateCaseMessageValueInModalSequence.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
+/* eslint-disable sort-keys-fix/sort-keys-fix */
```
Comment (kkoskelin) [inline web-client/src/presenter/sequences/updateCreateCaseMessageValueInModalSequence.js:1]: Any particular reason to suspend this `eslint` rule?

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**